full-node/src/network_service.rs

```diff
@@ -859,17 +855,23 @@ async fn background_task(mut inner: Inner) {
                             inner
                                 .peering_strategy
                                 .remove_address(expected_peer_id, remote_addr.as_ref());
-                            match inner
-                                .peering_strategy
-                                .insert_or_set_connected_address(&peer_id, remote_addr.clone().into_vec(), 10) // TODO: constant
+                            if let basic_peering_strategy::InsertAddressResult::Inserted {
+                                address_removed: Some(addr_rm),
+                            } = inner.peering_strategy.insert_or_set_connected_address(
+                                &peer_id,
+                                remote_addr.clone().into_vec(),
+                                10,
+                            )
+                            // TODO: constant
```

Comment:
```suggestion
                            if let basic_peering_strategy::InsertAddressResult::Inserted {
                                address_removed: Some(addr_rm),
                            } = inner.peering_strategy.insert_or_set_connected_address(
                                &peer_id,
                                remote_addr.clone().into_vec(),
                                10, // TODO: constant
                            )
```

Not sure if the suggestion passes rustfmt, but right now this TODO is detached from what it points to

Reply:
Modifying the TODO to be something like `// TODO: 10 is an arbitrary constant` also works if it's annoying

Reply:
Somewhere in the process I probably moved it. They are now fixed and I run `cargo (+nightly) fmt`